Balancer-Bots/Large-Internal-Balance-Deposit-Withdrawal/src/balance.fetcher.ts

```diff
@@ -0,0 +1,41 @@
+import { providers, Contract, BigNumber } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import LRU from "lru-cache";
+import { TOKEN_ABI } from "./constants";
+
```

Comment:
No need to change this now, it's totally fine. But using `SmartCaller` might have made things a bit easier (see https://github.com/NethermindEth/Forta-Agents/blob/%40balancer/large-swap/Balancer-Bots/Large-Swap/src/utils.ts).